nemo/lightning/pytorch/optim/megatron.py

```diff
@@ -90,10 +90,9 @@ def sharded_state_dict(
                 model_sharded_state_dict,
                 optimizer_state_dict=None,
                 is_loading=False,
-                # dist_ckpt_parallel_save=False, ## TODO: fix!
+                dist_ckpt_parallel_save=True,
```

Comment:
how can this be configured?

Reply:
I've made everything configurable from the strategy. Hopefully this is sufficient for now. If there are any concerns, we can work on a better design in a subsequent PR.